### PR TITLE
Fix a race condition in the benchmark example and …

### PR DESCRIPTION
**Pull request by anhu**
…make all output go to stderr.

### CODE REVIEW

**Review by dgarske — CHANGES_REQUESTED** [review]
Is the reason for piping to `fprintf(stderr,` so the messages can be filtered or captured differently? Interestingly our WOLFSSL_MSG also goes to stderr by default.

File: examples/benchmark/tls_bench.c [hunk 1]
```diff
@@ -747,14 +750,19 @@ static int SetupSocketAndConnect(info_t* info, const char* host,
      * Sets the socket to be stream based (TCP),
      * 0 means choose the default protocol. */
     if ((info->client.sockFd = socket(AF_INET, SOCK_STREAM, 0)) == -1) {
-        printf("ERROR: failed to create the socket\n");
+        fprintf(stderr, "ERROR: failed to create the socket\n");
         return -1;
     }
 
     /* Connect to the server */
+    while (info->serverListening == 0) {
```
[the remark this answers is not in the document]
Comment (anhu) [reply examples/benchmark/tls_bench.c:758]: ah...right....I remember we talked about that.  Ok.  Will do.

**Comment by dgarske** [comment]
Also with c99 and `usleep`. Consider using `XSLEEP_MS` macro.

```
./configure --enable-jobserver=4 CFLAGS="-std=c99" --enable-all
examples/benchmark/tls_bench.c:760:9: error: implicit declaration of function ‘usleep’; did you mean ‘sleep’? [-Werror=implicit-function-declaration]
```

**Comment by anhu** [comment]
Oh dear.... How embarrassing.  :blush: 
I'll get those fixed up.

**Review by dgarske — APPROVED** [review]

**Comment by anhu** [comment]
Woot!!!

**Pull request merged**